src/components/calcite-dropdown/calcite-dropdown.e2e.ts

```diff
@@ -656,25 +656,30 @@ describe("calcite-dropdown", () => {
       const trigger = await element.find("calcite-button[slot='dropdown-trigger']");
       const dropdownWrapper = await page.find(`calcite-dropdown >>> .calcite-dropdown-wrapper`);
       const calciteDropdownOpen = await element.spyOnEvent("calciteDropdownOpen");
+      const calciteDropdownClose = await element.spyOnEvent("calciteDropdownClose");
 
       expect(await dropdownWrapper.isVisible()).toBe(false);
       await trigger.click();
       await page.waitForChanges();
       expect(await dropdownWrapper.isVisible()).toBe(true);
+      await page.waitForChanges();
+      expect(calciteDropdownOpen).toHaveReceivedEventTimes(1);
```

Comment:
WDYT about asserting both events after each toggle action? I'll leave this up to you to decide. 😄